### DISCUSSION TITLE
Building .ipa in IntelliJ quits without error before finishing

### DISCUSSION THREAD

**Discussion by chrslgr** [airsdk/Adobe-Runtime-Support#3585] [Q&A]
Hi Air-Community.

I develop an Air app with release for all major platforms. 
Recently i switched to a new MacBook, and since then packaging the .ipa only creates .swf, .dsym etc, but not the .ipa.
I dont get any errors, the IntelliJ log doesnt show any errors too.
Has anyone ever had a problem like that? I cant find any hint in the net what the problem could be....

MacOS 15.1.1
IntelliJ 2024.1.3
Air SDK 51.1.2.3
iOS SDK 18.1

Thank you very much for your assistance.
Chris

**Comment by ajwfrost** [comment]
Hi

If you enable the ADT logging, it may provide some insight into what's happening... you can either just run the AIR SDK Manager in "troubleshooting" mode, or you can set the `DebugOut=true` value in the ~/.airsdk/adt.cfg file, and then check the ~/adt.log file after re-trying it..

thanks

**Comment by chrslgr** [comment]
thank you andrew! with the log i found out it is a certificate problem. after updating the certificate everything built perfectly.
thanks for all the work you put into the air runtime. it still works like a charm!!
chris
